popsycle/synthetic.py

```diff
@@ -768,9 +864,10 @@ def _make_comp_dict(iso_dir, log_age, currentClusterMass, star_dict, next_id,
             ########## 
             # Assign x, y, z position.
             ##########
-            comp_dict['px'], comp_dict['py'], comp_dict['pz'] = galactic_to_heliocentric(comp_dict['rad'],
-                                                                                         comp_dict['glat'],
-                                                                                         comp_dict['glon'])
+            comp_dict['px'], comp_dict['py'], comp_dict[
```

Comment:
what happened here? can you do this linebreak?

Reply:
fixed